llvm/include/llvm/CodeGen/GlobalISel/GenericMachineInstrs.h

```diff
@@ -558,6 +558,60 @@ class GVecReduce : public GenericMachineInstr {
   }
 };
 
+// Represents an integer binary operation.
+class GIBinOp : public GenericMachineInstr {
```

Comment:
Can we rename this to `GIntBinop` otherwise `GI` could be confused with *GI*Sel.